hasjob/templates/admin_dashboard.html.jinja2

```diff
@@ -12,7 +12,7 @@
   </div>
 {% endblock %}
 
-{% block footerscripts %}
+{% block tablelayoutscripts %}
```

Comment:
This should be `tablayout`, not `tablelayout`. Please amend in the source template as well.

Reply:
Fixed